src/QsCompiler/Tests.Compiler/TestCases/QirEntryPointTests/TestUnitReceivesBool.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "Name": "UseBoolArg",
+  "Arguments": [
+    { "Name": "BoolArg" }
+  ]
```

Comment:
Looks like the "Arguments" are not use by anybody. Only the "Name".
Is the "Arguments" section a plan for the future?

Reply:
The goal of this PR is just to setup a base to start the implementation of the C++ entry-point driver generator. It currently just uses the name of the entry-point operation to show an example but Arguments will be used to generate the rest of the C++ code. Sam will be working on this starting next week.